app/services/grow/grow-data.ts

```diff
@@ -0,0 +1,141 @@
+export const GROWTH_TIPS = () => [
+  {
+    title: 'Set Up Tips',
+    icon: 'icon-balance',
+    link: 'http://www.streamlabs.com',
+    cta: 'Set Up Tips',
+    description:
+      "Go to Donation Settings and enter your personal information to get started. If you're a new streamer, we recommend starting with PayPal.",
```

Comment:
Not sure how often we will be changing these, but just bringing up in case we want to add these to i18n as well.